sample-yamls/pod_rteval_loadgen.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rteval
+  annotations:
+    cpu-load-balancing.crio.io: "disable"
+    irq-load-balancing.crio.io: "disable"
+    cpu-quota.crio.io: "disable"
+spec:
+  runtimeClassName: performance-openshift-node-performance-profile
+  restartPolicy: Never
+  containers:
+  - name: rteval
+    image: quay.io/container-perf-tools/rteval
+    imagePullPolicy: Always
+    env:
+    - name: tool
+      value: "rteval"
+    # In this example we are generating load on only the first CPU  
+    - name: LOAD_CPUS
+      value: "0"
```

Comment:
CPU 0 is going to be one of the reserved platform CPUs. Do you want to run the load there? Also, in kubernetes you do not have control over which CPUs the pod is going to run on. Would it be better to create a guaranteed QoS pod and then just specify the number of LOAD_CPUS and have the cmd.sh script allocate that number of CPUs from the set that are assigned to the pod at creation time?